src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,26 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private double dozingWidth;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(double dozingWidth) {
+        this.dozingWidth = dozingWidth;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public double getDozingWidth() {
+        return dozingWidth;
+    }
+
+    public Bulldozer setDozingWidth(int dozingWidth) {
```

Comment:
The parameter type of 'setDozingWidth' should match the field type. The field 'dozingWidth' is of type 'double', but the parameter is an 'int'.